tests/shared/src/test/scala/org/http4s/StaticFileSuite.scala

```diff
@@ -114,6 +114,14 @@ class StaticFileSuite extends Http4sSuite {
     emptyFile.flatMap(StaticFile.fromPath[IO](_).value).map(_.isDefined).assert
   }
 
+  test("handle a symlink") {
+    StaticFile
+      .fromPath[IO](Path("tests/shared/src/test/resources/lipsum-symlink.txt"))
+      .semiflatMap(_.body.compile.count)
+      .getOrElse(0)
+      .assertEquals(24005)
```

Comment:
What's the magic number? Could we replace it with computation?

Reply:
Yes we can, we can just directly read the size of the file `lorem-ipsum.txt` instead of via the symlink.